.circleci/config.yml

```diff
@@ -70,3 +70,4 @@ workflows:
                 - "3.0.0"
                 - "3.1.1"
                 - "3.2.0"
+                - "3.3.0"
```

Comment:
We could drop Ruby 2.7 support while we're at it as 2.7 is EOL.

Reply:
The underlying rubocop version still supports it, so I think we might as well keep it as long as it doesn't hurt the maintenance of this gem. 

Dropping support for ruby 2.7 might actually make it harder to move a project from 2.7 to 3+, since instead of bumping rubocop, fixing things and then bumping ruby, you are forced to either bump both of them together or go through 2 rubocop version upgrades.